.gitmodules

```diff
@@ -3,5 +3,4 @@
 	url = https://github.com/nodejs/http-parser.git
 [submodule "vendor/llhttp"]
 	path = vendor/llhttp
-	url = https://github.com/nodejs/llhttp.git
-	branch = release
```

Comment:
Why drop the branch?

Reply:
I've added that branch config in the previous release as it helped me get the tag: release/v6.0.9
When i repeated the steps to update the submodule it got to tag: release/v2.1.6

Perhaps there are other solutions, but this was what i needed to do to get it to the tag release/v6.0.10
```
cd vendor\llhttp\
git checkout refs/tags/release/v6.0.10
```